src/test/jenkins/job/withCredentialsAndParameters.jenkins

```diff
@@ -0,0 +1,12 @@
+properties([
+    parameters([
+        booleanParam(name: 'myBooleanParam', description: 'My boolean typed parameter'),
```

Comment:
This is unused in the pipeline script. Should it be echoed or evaluated or something?